cloud/scope/powervs_cluster.go

```diff
@@ -156,6 +167,13 @@ func NewPowerVSClusterScope(params PowerVSClusterScopeParams) (*PowerVSClusterSc
 		options.Zone = *params.IBMPowerVSCluster.Spec.Zone
 	}
 
+	// Fetch the PowerVS service endpoint.
+	powerVSSvcEndpoint := endpoints.FetchEndpoints(string(endpoints.PowerVS), params.ServiceEndpoint)
```

Comment:
can we rename `powerVSSvcEndpoint` to something like - `pvsEndpoint` or `pvsServiceEndpoint` or `powerVSServiceEndpoint` or `powerVSEndpoint`

Reply:
Renamed as suggested.